nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/frontend/package.json

```diff
@@ -20,7 +20,7 @@
     "preinstall": "npm --ignore-scripts install npm-force-resolutions && npm-force-resolutions"
   },
   "dependencies": {
-    "angular": "1.8.2",
+    "angular": "1.8.3",
     "angular-animate": "1.8.2",
```

Comment:
FYI, these other angular-* packages should also be updated when upgrading angular.